src/com/mraof/minestuck/item/MinestuckItems.java

```diff
@@ -207,7 +213,7 @@ public EntityHanging createEntity(World worldIn, BlockPos pos, EnumFacing facing
 			return new EntityShopPoster(worldIn, pos, facing, stack, meta);
 		}
 	}.setUnlocalizedName("shopPoster").setMaxStackSize(1).setCreativeTab(TabMinestuck.instance);
-	
+
```

Comment:
Looks like you've got the setting to not keep indents on empty lines. Since you use intellij, go to File/Settings... and there Editor/Code Style/Java and check "Keep indents on empty lines". You don't have to bother with adding the indents back, but it should prevent it in the future.